.github/workflows/build.yml

```diff
@@ -0,0 +1,85 @@
+name: Build and test
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+  release:
+    types: 
+      - created
+jobs:
+  build-and-test:
+    strategy:
+      matrix:
+        docker-version: [19.03]
+    runs-on: ubuntu-latest
+    name: Build and run Docker images
+    steps:
+      - name: Update Packages
+        run: sudo apt-get update -yqq --fix-missing
+      # Difference here is remote-docker-setup in CircleCI vs Docker install in GH action
+      # In CircleCI, When setup_remote_docker executes, a remote environment will be created, 
+      # and your current primary container will be configured to use it. 
+      # Then, any docker-related commands you use will be safely executed in this new environment.
+
+      # In GH action, we can setup remote docker but then we have to host our own docker setup somewhere and SSH into that. 
+      # Installing docker every time eats a bit of a time at the moment but 
+      - name: Install Docker
+        uses: docker-practice/actions-setup-docker@master
```

Comment:
can we have a fixed version for this action `docker-practice/actions-setup-docker@master` ?